components/NewPersonView/Layout.tsx

```diff
@@ -0,0 +1,161 @@
+import Head from 'next/head';
+import Link from 'next/link';
+import { format } from 'date-fns';
+import { useRouter } from 'next/router';
+import { Allocation, Resident, User } from 'types';
+import s from './index.module.scss';
+import { useRelationships } from 'utils/api/relationships';
+import { useAllocatedWorkers } from 'utils/api/allocatedWorkers';
+import React from 'react';
+import WarningNotes from 'components/WarningNote/WarningNotes';
+import { useAuth } from 'components/UserContext/UserContext';
+import { canManageCases } from 'lib/permissions';
+
+interface NavLinkProps {
+  href: string;
+  children: React.ReactChild;
+}
+
+const NavLink = ({ href, children }: NavLinkProps) => {
+  const router = useRouter();
+  return (
+    <li>
+      <Link href={href}>
+        <a
+          className={`lbh-link lbh-link--no-visited-state ${
+            router.asPath === href && s.navLinkActive
+          }`}
+        >
+          {children}
+        </a>
+      </Link>
+    </li>
+  );
+};
+
+interface Props {
+  person: Resident;
+  children: React.ReactChild;
+}
+
+const summariseAllocations = (allocations: Allocation[]): string | null => {
+  if (allocations?.length === 1)
+    return ` · Allocated to ${allocations[0].allocatedWorker}`;
+  if (allocations?.length === 2)
+    return ` · Allocated to ${allocations[0].allocatedWorker} and 1 other`;
+  if (allocations?.length > 1)
```

Comment:
I know the previous if statement would already have returned but I think saying `if (allocations?.length > 2)` would make more sense.